HelpSource/Classes/Integer.schelp

```diff
@@ -213,9 +250,75 @@ code::
 23.0.factorial // for larger factorials, use floats.
 ::
 
+method:: fold
+Folds in to a value between code::lo:: and code::hi::.
+returns:: an Integer between code::lo:: and code::hi::.
+discussion::
+code::
+(0..12).fold(0,3); // returns [ 0, 1, 2, 3, 2, 1, 0, 1, 2, 3, 2, 1, 0 ]
+::
+
+method:: log2Ceil
+returns:: next larger integer of the base 2 logarithm of the receiver.
+discussion::
+code::
+17.log2; // returns 4.0874628412503
+17.log2Ceil; // returns 5
+17.log2.ceil.asInteger; // equivalent to the previous line
+::
+
+method:: wrap
+Wraps in to a value between code::lo:: and code::hi::.
+returns:: an Integer between code::lo:: and code::hi::.
+discussion::
+code::
+(0..12).wrap(0,3); // returns [ 0, 1, 2, 3, 0, 1, 2, 3, 0, 1, 2, 3, 0 ]
+::
 
 subsection:: Misc
 
+method::archiveAsCompileString
```

Comment:
This method is part of an interface that is not documented for any class. Maybe lik to code::Classes/Archive:: ?

(but I don't mind).